Makefile

```diff
@@ -1,3 +1,8 @@
+WORKING_DIR := recommender
+EXTRACT_DIR := $(WORKING_DIR)/translations/en/LC_MESSAGES
```

Comment:
it should be in `conf/locale` as far as I know. check OEP-58 for more info.

Reply:
Fixed. Please note that I had to edit `.tx/config` to do that